### DISCUSSION TITLE
Twister UPDATE BOX86 failure?

### DISCUSSION THREAD

**Discussion by Lander1979** [ptitSeb/box86#461] [Q&A] [answered]
I am using a standard 32 bit Twister OS install on a rPi 4b 4Gig. The OS BOOT/SYSTEM is completely located on a Toshiba 120Gig USB3 SSD, no Flash MMC cards are in use.

When I run Twister OS Diagnostics, it says;
[Box86 with Dynarec v0.2.3 0e54aaf7 built on sep 6 2021 11:10:11]
          GL4ES found
          Binfmt-magic present: Yes
          interpreter /usr/local/bin/box86
          Latest commit:e30eae76527e24fdd9d208768406c8f77099e717
          Box86 is out of date. If you are having issues,
          consider updating Box86 using: Menu > Games > UPDATE BOX86

When I run UPDATE BOX86 a terminal briefly appears and quickly scrolls some white text and it promptly closes, no actual update is performed.  Is it broken or was the update not for my 32 bit system and therefore not applied?  I'm not sure wether I should just leave it or start panicking and try to reinstall it?

**Comment by ptitSeb** [comment, marked as answer]
There hasn't been code update since the 6th of setembre. The next updates were on doc and CI, so not directly code. So I think you are ok.

**Reply by Lander1979** [reply]
Thankyou for the quick reply.  Also, a Huge Thankyou for Box86, It is the holy grail to ARM based Linux.
